src/data/externalTutorials.json

```diff
@@ -1,13 +1,98 @@
 [
+  {
+    "url": "https://youtu.be/wUjYK5gwNZs",
+    "title": "Learn Solidity, Blockchain Development, & Smart Contracts | Powered By AI - Full Course (12 - 15)",
+    "description": "This course will give you a full introduction to all of the core concepts related to blockchain, smart contracts, Solidity, ERC20s, Foundry, DeFi, Chainlink, Ethereum, upgradable smart contracts, DAOs, IPFS, and more. Follow along with the videos and you'll be a blockchain wizard in no time!",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": [
+      "solidity",
+      "smart contracts",
+      "video",
+      "oracles",
+      "NFT",
+      "ERC-20",
+      "foundry",
+      "mocking",
+      "testing",
+      "defi",
+      "openzeppelin",
+      "slither",
+      "storage",
+      "alchemy",
+      "block explorer",
+      "fuzzing"
+    ],
+    "skillLevel": "beginner",
+    "timeToRead": "495",
+    "lang": "en",
+    "publishDate": "06/07/2023"
+  },
+  {
+    "url": "https://youtu.be/sas02qSFZ74",
+    "title": "Learn Solidity, Blockchain Development, & Smart Contracts | Powered By AI - Full Course (7 - 11)",
+    "description": "This course will give you a full introduction to all of the core concepts related to blockchain, smart contracts, Solidity, ERC20s, Foundry, DeFi, Chainlink, Ethereum, upgradable smart contracts, DAOs, IPFS, and more. Follow along with the videos and you'll be a blockchain wizard in no time!",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": [
+      "solidity",
+      "smart contracts",
+      "video",
+      "oracles",
+      "NFT",
+      "ERC-20",
+      "foundry",
+      "mocking",
+      "testing",
+      "defi",
+      "openzeppelin",
+      "slither",
+      "storage",
+      "alchemy",
+      "block explorer",
+      "fuzzing"
+    ],
+    "skillLevel": "beginner",
+    "timeToRead": "640",
+    "lang": "en",
+    "publishDate": "06/07/2023"
+  },
+  {
+    "url": "https://youtu.be/umepbfKp5rI",
+    "title": "Learn Solidity, Blockchain Development, & Smart Contracts | Powered By AI - Full Course (0 - 6)",
+    "description": "This course will give you a full introduction to all of the core concepts related to blockchain, smart contracts, Solidity, ERC20s, Foundry, DeFi, Chainlink, Ethereum, upgradable smart contracts, DAOs, IPFS, and more. Follow along with the videos and you'll be a blockchain wizard in no time!",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": [
+      "solidity",
+      "smart contracts",
+      "video",
+      "oracles",
+      "NFT",
+      "ERC-20",
+      "foundry",
+      "mocking",
+      "testing",
+      "defi",
+      "openzeppelin",
+      "slither",
+      "storage",
+      "alchemy",
+      "block explorer",
+      "fuzzing"
```

Comment:
@PatrickAlphaC You've been busy! Thanks a lot of posting this, looks like a great resource!

Any chance we could refine these a little? Been actively trying to reduce the tag-fatigue on this page... It looks like we've repeated the same list of tags for all three blocks—could we possibly zero-in on which ones apply most to each of the three different segments?

@minimalsm @corwintines Remind me, but I also recall us trying to reduce the number of products used as tags, but I see a few of these are still used on others tutorials, ie. alchemy or openzeppelin

Otherwise this looks great!

Reply:
yess.... uh... So it's a 27-hour course and goes over... everything. Which ones do you think would be best to remove?

Reply:
@PatrickAlphaC At a glance, it looks like these three videos should be tackled sequentially to get the most from them. We don't need to link to all three on our tutorials page; if someone does the first eight hours, they won't be circling back to ethereum.org to find the link to the next one, and someone going directly to 2/3 or 3/3 is a bad experience.

We should remove parts 7-11 and 12-15 and generalise the title of parts 0-6.

> @minimalsm @corwintines Remind me, but I also recall us trying to reduce the number of products used as tags, but I see a few of these are still used on others tutorials, ie. alchemy or openzeppelin

Let's just leave the tags discussion for now, as we haven't optimised this across the board yet.

Reply:
Wow I'm so sorry I didn't see these comments till now!